Updating billing page (#3916)
fishtown-analytics/docs.getdbt.com: website/docs/docs/cloud/billing.md

### Developer plan billing

> Developer plans are free and include one Developer license and 3,000 models each month. Models are refreshed at the beginning of each calendar month. If you exceed 3,000 models, any subsequent runs will be canceled until models are refreshed or until you upgrade to a paid plan. The rest of the dbt Cloud platform is still accessible, and no work will be lost.

Reviewer: ```suggestion
Developer plans are free and include one Developer license and 3,000 models each month. Models are refreshed at the beginning of each calendar month. If you exceed 3,000 models, any subsequent runs will be canceled until models are refreshed or until you upgrade to a paid plan. 
The rest of the dbt Cloud platform is still accessible, and you won't lose any of your work.
```

> All included successful models built numbers above reflect our most current pricing and packaging. Based on your usage terms when you signed up for the Developer Plan, the included model entitlements may be different from what’s reflected above.

Reviewer: ```suggestion
The number of all included successful models built mentioned above reflects our most current pricing and packaging. Depending on the usage terms you agreed to when you signed up for the Developer Plan, the included model entitlements may be different from what’s reflected above.
```

### Team plan billing 

Team customers pay monthly via credit card for seats and usage, and accounts include 15,000 models monthly. Seats are charged upfront at the beginning of the month. If you add seats during the month, seats will be prorated and charged on the same day. Seats removed during the month will be reflected on the next invoice and are not eligible for refunds. You can change the credit card information and the number of seats from the billings section anytime. Accounts will receive one monthly invoice that includes the upfront charge for the seats and the usage charged in arrears from the previous month.

Usage is calculated and charged in arrears for the previous month. If you exceed 15,000 models in any month, you will be billed for additional usage on your next invoice. Additional use is billed at the rates on our [pricing page](https://www.getdbt.com/pricing). 

Included models that are not consumed do not roll over to future months. You can estimate your bill with a simple formula:

`($100 x number of developer seats) + ((models built - 15,000) x $0.01)`

> All included successful models built numbers above reflect our most current pricing and packaging. Based on your usage terms when you signed up for the Team Plan, the included model entitlements may be different from what’s reflected above.

Reviewer: ```suggestion
The number of all included successful models built mentioned above reflects our most current pricing and packaging. Depending on the usage terms you agreed to when you signed up for the Developer Plan, the included model entitlements may be different from what’s reflected above.
```
